ST_main.c

```diff
@@ -67,10 +67,11 @@ SW_FILE_STATUS SW_File_Status;
   void stat_Output_AllMorts( void) ;
   void stat_Output_AllBmass(void) ;
 
-  void runGrid( void ); //for the grid... declared in ST_grid.c
+  /* void runGrid( void ); //for the grid... declared in ST_grid.c */
 
```

Comment:
@kpalmqui I commented out `runGrid` because I commented out the entirety of the gridded code and therefore, I wanted to avoid compiler warnings from having a function declaration without a definition. Let me know if this needs further explaining.

Reply:
@kpalmqui Also, the reason why I commented out the entirely of the gridded code was because of changes from #225, which break the gridded code. Combined with what @chaukap is already doing with all the variables in the gridded code, it would be best to fix these issues in that branch.

Reply:
@fdpierson thanks for the additional explanation here. It does make sense to resolve issues with the gridded code in the gridded_code branch, despite it not being ideal to merge commented out code into the supposedly stable master. Guess that is the only possible way forward.